### ISSUE TITLE
No Aria-label in create pages edit paragraph

### ISSUE THREAD

**Issue by AfLosada** [AfLosada/Ghost_MISO#27]
**Describe the bug**
No Aria-label in create pages edit paragraph. This should be enabled to facilitate navigation for users with screen-readers.

**To Reproduce**
1. Create Page
2. Click on paragraph
3. Open inspect tools
4. See that it has no aria-label

**Expected behavior**
Every editable element should have an aria-label that should help screen-readers detect it

**Screenshots**
[VIDEO](https://youtu.be/SRWVK4kEcxM)

**Environment (please complete the following information):**
 - Device: DESKTOP
 - OS: Windows 11¿
 - Browser: Chrome
 - Version 5.68.0
